src/offset/fixed.rs

```diff
@@ -154,9 +154,7 @@ impl fmt::Display for FixedOffset {
 impl arbitrary::Arbitrary<'_> for FixedOffset {
     fn arbitrary(u: &mut arbitrary::Unstructured) -> arbitrary::Result<FixedOffset> {
         let secs = u.int_in_range(-86_399..=86_399)?;
-        let fixed_offset = FixedOffset::east(secs)
-            .expect("Could not generate a valid chrono::FixedOffset. It looks like implementation of Arbitrary for FixedOffset is erroneous.");
-        Ok(fixed_offset)
+        Ok(FixedOffset::east(secs).unwrap())
```

Comment:
Could the unwrapping be prevented by mapping the error? Untested: `FixedOffset::east(secs).or(Err(arbitrary::Error))`

Reply:
The range we supply to `arbitrary::Unstructured::int_in_range` is exactly equal to the range `FixedOffset` supports, and I don't see that changing. This should simply never fail, so I even considered the `expect` to be overkill.